### ISSUE TITLE
Creating a list with all personal bests

### ISSUE THREAD

**Issue by LinusFresz** [Moony22/prisma#17]
I would like to request a feature with the following attributes:
The feature should be able to show all personal bests you have made in the different events in Prisma. 
This could be look like:
| Event     | Single    | mo3   | avg5  | avg12     | avg50     | avg100    | global average    |
|-------    |--------   |------ |------ |-------    |-------    |--------   |----------------   |
| 2x2x2     | 1         | 1.5   | 2     | 2.5       | ...       |           |                   |
| 3x3x3     | 10        | 10.5  | 11    | ...       |           |           |                   |
| ...       |           |       |       |           |           |           |                   |

(well.. maybe a better alignment for the columns)
So basically just a table with all your pbs.

Regards,
Linus Fresz
